gnd/src/main/java/com/google/android/gnd/persistence/local/room/AreaEntity.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.gnd.persistence.local.room;
+
+import androidx.annotation.NonNull;
+import androidx.room.ColumnInfo;
+import androidx.room.Entity;
+import androidx.room.PrimaryKey;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.LatLngBounds;
+import com.google.android.gnd.model.basemap.Area;
+import com.google.auto.value.AutoValue;
+
+/** Represents a {@link Area} in the local data store. */
+@AutoValue
+@Entity(tableName = "area")
```

Comment:
Naming nit: Looking at the db and the class name in isolation "Area" may be too general - should we call them `OfflineAreaEntity` and `offline_area` instead for clarity?

Reply:
sg!